pkg/build/api/v1/defaults_test.go

```diff
@@ -17,22 +17,23 @@ import (
 func TestDefaults(t *testing.T) {
 	testCases := []struct {
 		External runtime.Object
-		Internal runtime.Object
 		Ok       func(runtime.Object) bool
 	}{
 		{
 			External: &v1.Build{
 				Spec: v1.BuildSpec{
 					CommonSpec: v1.CommonSpec{
 						Strategy: v1.BuildStrategy{
-							Type: v1.DockerBuildStrategyType,
+							DockerStrategy: &v1.DockerBuildStrategy{},
```

Comment:
Fixed in https://github.com/ncdc/origin/commit/8fc60565c14a83c6f57151f9018fbae70754822d